pkg/kubectl/cmd/util/helpers.go

```diff
@@ -597,6 +597,19 @@ func ManualStrip(file []byte) []byte {
 	return stripped
 }
 
+// GetObjectName receives a resource Info and returns an approximate
+// form of the resource's kind/name.
+func GetObjectName(info *resource.Info) string {
```

Comment:
any reason not to make this a method on resource.Info, similar to it's existing `String()` method?

Reply:
I'll open a followup to address that.